styx-common/src/main/java/com/spotify/styx/api/ServiceAccountUsageAuthorizer.java

```diff
@@ -199,6 +203,10 @@ private ServiceAccountUsageAuthorizationResult authorizationCheck(WorkflowId wor
       return ServiceAccountUsageAuthorizationResult.builder()
           .serviceAccountProjectId(projectId)
           .accessMessage(firstPresent(
+              // Check if the principal is an admin
+              () -> memberStatus(principalEmail, administrators)
```

Comment:
If we configure `group:...` for staging, this will always result RuntimeException because of missing proper permission, right?

Reply:
Yeah

Reply:
So we configure individuals then?